lib/Core/Tests/Input/StringQueryInputTest.php

```diff
@@ -319,8 +319,10 @@ public function provideEmptyInputTests()
     public function provideSingleValuePairTests()
     {
         return [
-            ['name: value, value2, ٤٤٤٦٥٤٦٠٠, 30, 30L, !value3;'],
-            ['name: value, value2, ٤٤٤٦٥٤٦٠٠, 30, 30L, !value3'],
+            ['name: value, value2, ٤٤٤٦٥٤٦٠٠, 30, 30L, !value3; @date: uP', ['@date' => 'ASC']],
+            ['name: value, value2, ٤٤٤٦٥٤٦٠٠, 30, 30L, !value3; @date: Up', ['@date' => 'ASC']],
+            ['name: value, value2, ٤٤٤٦٥٤٦٠٠, 30, 30L, !value3;', ['@date' => 'DESC', '@id' => 'ASC']],
+            ['', ['@date' => 'DESC', '@id' => 'ASC']],
```

Comment:
Default sort options still get applied, even if the search & filter part is empty.